From 967cb712a0f7305001f813e4350e792e2086790e Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 21 Mar 2014 11:51:13 +0800
Subject: [PATCH 1047/1074] ENGR00304827 net: fec: init the phy_id to invalid
 address

Init the phy_id to invalid address in .probe() function.

Signed-off-by: Fugang Duan <B38611@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 drivers/net/ethernet/freescale/fec_main.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fec_main.c b/drivers/net/ethernet/freescale/fec_main.c
index 5f82032..ca82dfa 100644
--- a/drivers/net/ethernet/freescale/fec_main.c
+++ b/drivers/net/ethernet/freescale/fec_main.c
@@ -2574,6 +2574,7 @@ fec_probe(struct platform_device *pdev)
 	platform_set_drvdata(pdev, ndev);
 
 	fec_of_init(pdev);
+	fep->phy_id = PHY_MAX_ADDR;
 	of_property_read_u32(pdev->dev.of_node, "phy-id", &fep->phy_id);
 	ret = of_get_phy_mode(pdev->dev.of_node);
 	if (ret < 0) {
-- 
1.7.5.4

